ha_mroonga.cpp

```diff
@@ -14978,31 +15076,33 @@ double ha_mroonga::wrapper_read_time(uint index, uint ranges, ha_rows rows)
   }
   DBUG_RETURN(res);
 }
-#endif
+#  endif
 
 double ha_mroonga::storage_read_time(uint index, uint ranges, ha_rows rows)
 {
   MRN_DBUG_ENTER_METHOD();
-  double time = handler::read_time(index, ranges, rows);
+  double time =
+    handler::read_time(index, ranges, rows);
```

Comment:
Is it automatically changed by the formatter?

Reply:
Ah...
No. This is my mistake.

Reply:
I've fixed it.